src/Common/Repositories/Abstract/IRobotRepository.cs

```diff
@@ -6,8 +6,8 @@ public interface IRobotRepository : IBaseRepository<RobotModel>
     {
         Task<RobotModel> PatchRobotAsync(Guid id, RobotModel patch);
 
-        Task<List<Guid>> GetConnectedEdgesIdsAsync(Guid robotId);
+        Task<List<EdgeModel>> GetConnectedEdgesIdsAsync(Guid robotId);
```

Comment:
Are we returning the `EdgeModel` or ids? The name is very missleading... 
Should be `GetConnectedEdgesAsync` I assume.